docs/modules/reference/pages/service-assurance/monitors/TrivialTimeMonitor.adoc

```diff
@@ -0,0 +1,68 @@
+
+= TrivialTimeMonitor
+
+TrivialTimeMonitor connects to a server that supports the Time Protocol and sends the time as a 32-bit unsigned integer in binary format and in network byte order, representing the number of seconds since 00:00 (midnight) 1 January, 1900 GMT, and fails if the defined clock skew is too large.
+
+== Monitor facts
+
+[cols="1,7"]
+|===
+| Class Name
+| `org.opennms.netmgt.poller.monitors.TrivialTimeMonitor`
+|===
+
+== Configuration and use
+
+.Optional monitor-specific parameters for the TrivialTimeMonitor
+[options="header"]
+[cols="1,3,2"]
+|===
+| Parameter
+| Description
+| Default
+
+| protocol
+| Default layer-4 protocol to use.
+| tcp
+
+| port
+| The destination port.
+| 37
+
+| allowedSkew
+| Default permissible skew in seconds between the remote and local clocks.
+| 30
+
+| retry
+| Number of attempts to get a response.
+| 0
+
+| timeout
+| Time in milliseconds to wait for a response.
+| 3000
+|===
+
+This monitor implements the <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
```

Comment:
For consistency, please move to below the heading (line 15).

Reply:
Ok. We'll have to do a sweep at some time to standardize. I noticed the detectors are inconsistent as well. Not critical right now.